packages/b/bootstrap-material-design-icons.json

```
@@ -10,7 +10,7 @@
     "web-font",
     "bootstrap"
   ],
-  "license": "(MIT AND CC-BY-4.0)",
+  "license": "(MIT OR CC-BY-4.0)",
```

```suggestion
  "license": "(MIT AND CC-BY-4.0)",
```
`AND` is a valid SPDX joiner, and makes sense here given there are icons that fall under CC, not MIT.